.eslintrc.js

```diff
@@ -5,7 +5,6 @@ module.exports = {
     'plugin:yml/standard'
   ],
   rules: {
-    'yml/plain-scalar': 'off',
```

Comment:
Is this for bonus points or actually necessary to fix?

Reply:
Not necessary, just restored the default setting to not overcomplicate it.

Reply:
From Slack: bonus points.